android/src/main/java/com/idehub/Billing/InAppBillingBridge.java

```diff
@@ -218,6 +218,22 @@ public void isPurchased(final String productId, final Promise promise){
         }
     }
 
+    @ReactMethod
+    public void isValidTransactionDetails(final String productId, final Promise promise){
+        if (bp != null) {
+            TransactionDetails details = bp.getPurchaseTransactionDetails(productId);
+            if (details != null && productId.equals(details.purchaseInfo.purchaseData.productId))
+            {
+                  boolean isValid = bp.isValidTransactionDetails(details);
+                  promise.resolve(isValid);
+            } else {
+                promise.reject("EUNSPECIFIED", "Failed to validate transaction details: " + ex.getMessage());
```

Comment:
I don't think this line will compile, `ex.getMessage()` references something which is not there. 

I'd also prefer if you wrap it in try/catch as per my comment: https://github.com/idehub/react-native-billing/issues/96#issuecomment-363669732

That will also give you the Exception `ex`.

Reply:
yea i copied your version into mine, I never had the error messaging in the log before